docs/reference/cli/app/list.mdx

```diff
@@ -10,13 +10,13 @@ The `app list` command lists all installed apps.
 ## Usage
 
 ```bash
-apps list
```

Comment:
FYI: `apps` is an alias for `app` - both work.
In this case, `apps list` might sound more natural than `app list`